src/main/java/seedu/duke/PairingList.java

```diff
@@ -120,6 +120,21 @@ public boolean isAlreadyPaired(Client client, Property property) {
         return false;
     }
 
+    /**
+     * Returns true if the rental price of the property exceeds the client's budget.
+     *
+     * @param client Client to be paired with the property.
+     * @param property Property to be paired with the client.
+     * @return True is rental price exceeds client's budget; false if rental price is equal to or lower than clien's
+     *         budget.
+     */
+
```

Comment:
Take note of the space here. After the Javadocs, there should not be an empty space.